src/scenes/game-scene.ts

```diff
@@ -20,6 +24,9 @@ class GameScene extends Scene {
     const { preLoad } = parallax(this);
     preLoad();
 
+    this.map = new Map(this);
```

Comment:
we are not suppose to initialise the `Map` here, only preload it

the `Map` must be in initialised in this Scene's constructor